src/js/engagement_view/src/components/engagementView/sidebar/tables/lensTable/lensTable.tsx

```diff
@@ -71,6 +73,8 @@ export const lensTable = (
                                     lens_type={lens.lens_type}
                                     score={lens.score}
                                     setLens={setLens}
+                                    selectedId={selectedIdState}
+                                    setLensTableState={setSelectedIdState}
```

Comment:
there's an incongruency in these args and their values, could we maybe sync them up a bit more if it makes sense to do so?

Reply:
resolved in different comment